src/lucky_cli/task.cr

```diff
@@ -7,6 +7,19 @@ abstract class LuckyCli::Task
     def name
       "{{@type.name.gsub(/::/, ".").underscore}}"
     end
+
+    def help_message
+      "Run this task with `lucky #{name}`"
+    end
+
+    def print_help_or_call(args : Array(String))
+      if args.any? { |arg| ["--help", "-h", "help"].includes?(arg) }
+        STDERR.puts help_message
+        help_message
```

Comment:
I think this can be removed  and the return type set to `Nil` so the result of this call is not used. That may mean reworking the tests to pass in an optional `io` argument.

```crystal
def print_help_or_call(args : Array(String), io : IO = STDERR) : Nil
  io.puts help_message
end

# in the tests
io = IO::Memory.new

LuckyCli::Runner.new
  .print_help_or_call(args, io: io)
io.to_s.should have_helper_message
```

Reply:
oh that's interesting. I can dig it!